network/network_windows.go

```diff
@@ -318,6 +323,36 @@ func (nm *networkManager) configureHcnNetwork(nwInfo *NetworkInfo, extIf *extern
 	return hcnNetwork, nil
 }
 
+func (nm *networkManager) addIPv6DefaultRoute() error {
+	var (
+		err error
+		out string
+	)
+
+	logger.Info("Adding default ipv6 route to windows node")
```

Comment:
Only log that the route is being added if it's missing. This would make it seem like the route is always being added on every CNI invocation.